modules/aws-cloudfront-security-headers/main.tf

```diff
@@ -81,17 +82,21 @@ data "archive_file" "this" {
 }
 
 resource "aws_lambda_function" "this" {
-  function_name    = var.name
-  description      = var.description
-  filename         = data.archive_file.this.output_path
-  source_code_hash = data.archive_file.this.output_base64sha256
-  handler          = "index.handler"
-  runtime          = var.runtime
-  role             = aws_iam_role.execution_role.arn
-  timeout          = var.timeout
-  memory_size      = var.memory_size
-  publish          = true
-  tags             = var.tags
+  function_name                  = var.name
+  description                    = var.description
+  filename                       = data.archive_file.this.output_path
+  source_code_hash               = data.archive_file.this.output_base64sha256
+  handler                        = "index.handler"
+  runtime                        = var.runtime
+  role                           = aws_iam_role.execution_role.arn
+  timeout                        = var.timeout
+  memory_size                    = var.memory_size
+  publish                        = true
+  tags                           = var.tags
+  reserved_concurrent_executions = 100
+  tracing_config {
+    mode = "Active"
+  }
```

Comment:
lets have reserved_concurrent_executions and tracing_config configurable from variables and use defaults with empty values as not always we need reserved concurrency and not always we need tracing be active